bin/sdk-cli.js

```diff
@@ -2,22 +2,87 @@
 
 /** @format */
 
+/**
+ * External dependencies
+ */
 const chalk = require( 'chalk' );
 const path = require( 'path' );
 const spawnSync = require( 'child_process' ).spawnSync;
+const yargs = require( 'yargs' );
 
 const buildBlockScript = path.resolve( __dirname, 'create-scripts/block.js' );
-const [ /* node */, /* bin/sdk-cli.js */, task, ...args ] = process.argv;
+const extensionsDir = path.resolve( __dirname, '../client/gutenberg/extensions' );
+const prePackagedBlocks = [
+	'editor-notes',
+	'hello-dolly',
+];
 
-if ( ! args.length || task !== 'build-block' ) {
-	console.log( chalk.red( 'usage: npx calypso-gutenberg-sdk build-block <block>' ) );
-	process.exit(1);
-}
+const buildBlock = argv => {
+	let entryFile;
 
-spawnSync( 'node', [ buildBlockScript, ...args ], {
-	env: {
-		SKIP_FLAG_IMAGES: true,
-	},
-	shell: true,
-	stdio: 'inherit',
-} );
+	if ( argv.block ) {
+		switch ( argv.block ) {
+			case 'editor-notes':
+				entryFile = path.join( extensionsDir, 'editor-notes/index.js' );
+				break;
+			case 'hello-dolly':
+				entryFile = path.join( extensionsDir, 'hello-dolly/hello-block.js' );
+				break;
+			default:
+				console.log( chalk.red( `Unknown block "${ argv.block }" - list available blocks with "list-blocks"` ) );
+				process.exit( 1 );
+		}
+	} else if ( argv.editorJs ) {
+		entryFile = argv.editorJs;
+	} else {
+		yargs.showHelp();
+		console.log( chalk.red( 'Missing a block entryfile.' ) );
+		process.exit( 1 );
+	}
+	spawnSync( 'node', [ buildBlockScript, entryFile, ( argv.outputDir || '' ) ], {
```

Comment:
In a follow-up PR we should import the build script as a function instead of calling it here.